stdlib/char.mli

```diff
@@ -52,6 +52,11 @@ val uppercase_ascii : char -> char
    using the US-ASCII character set.
    @since 4.03.0 *)
 
+val bits : char -> int
+(** Return the number of bits set to 1 in the bnary representation of the
```

Comment:
Typo "bnary".  Also, I'm not sure about the name.  For instance, it has nothing to do with Int64.bits_of_float.  Common names for the operation: Hamming weight, popcount.

Reply:
... or with `Random.bits`.

My vote goes to `popcount`.